app/models/competition.rb

```diff
@@ -188,6 +189,9 @@ class Competition < ApplicationRecord
   MAX_MARKDOWN_LENGTH = 255
   MAX_COMPETITOR_LIMIT = 5000
   MAX_GUEST_LIMIT = 100
+  NEWCOMER_MONTH_ENABLED = true
```

Comment:
Do we really need to hard-code this? We could just leave the form field as `enabled` all the time and have WCAT manually verify.

At least until we come up with a smarter solution of "marking" newcomer month start and end dates in the DB

Reply:
You marked this conversation as resolved but the constant is still there. It is completely unused as far as I can tell.